docs/preview/continuous-patching/README.md

```diff
@@ -0,0 +1,477 @@
+
+Special Suffix Rules
+
+- -1 to -999: These are considered patch tags.
+- -x where x > 999: These are not interpreted as patch tags; instead, that entire suffix is treated as part of the original tag. (Example: ubuntu:jammy-20240530 is considered an original tag, not a patched one.)
+This means if you push a new tag ending in -1 to -999 by accident, Continuous Patching will treat it like a patched image. We recommend you to avoid pushing tags that you want patched with the suffix -1 to -999.
+
+### Floating Tagging
+
+How it works
+
+A single mutable tag, -patched, will always reference the latest patched version of your image. For instance, if your base image tag is python:3.11, one patch creates python:3.11-patched. Subsequent patches will keep overwriting that same -patched tag with new digests.
+
+  
+![PatchingTimelineExample](./media/patching_timeline_example1.png)
+
+### Which Should I Use?
+
+Incremental (**default**): Great for environments where auditability and rollbacks are critical, since each new patch is clearly identified with an unique tag.
+
+Floating: Ideal if you prefer a single pointer to the latest patch for your CI/CD pipelines. Reduces complexity by removing the need to update references in downstream applications per patch, but sacrifices strict versioning, making it difficult to rollback. 
+
+
+## Installing the Continuous Patching Workflow
+
+Download the [private CLI extension](https://acrcssc.z5.web.core.windows.net/acrcssc-1.1.1rc6-py3-none-any.whl) for Continuous patching 
+
+Run the following command:
+```sh
+az extension add --source <path to the CLI extension>
+```
+
+## Enable the Continuous Patching Workflow
+To enable Continuous Patching, follow the series of steps below that outline the CLI process. These guidelines detail the lifecycle of a continuous patching workflow, from its creation to subsequent updates to eventual deletion. 
+1. Login to Azure CLI with az login
+```sh
+az login
+```
+2. Login to ACR 
+```sh
+az acr login -n <myRegistry>
+```
+3. Use your preferred text editor to create a file named continuouspatching.json. Linux example below
+```sh
+vim continuouspatching.json //any file creation command and file name will do
+```
+Copy in the JSON Schema:
+```sh
+{
+    "version": "v1",
+    "tag-convention" : "<incremental|floating>",
+    "repositories": [{
+        "repository": "<Repository Name>",
+        "tags": ["<comma-separated-tags>"],   
+        "enabled": <true|false>
+    }] 
+}
+```
+The schema ingests specific repositories and tags in an array format. Each variable is defined below:
+
+- "version" allows the ACR team to track what schema version you’re on. Do not change this variable unless instructed to.
+- "tag-convention" this is an optional field. Allowed values are "incremental" or "floating" - refer to **Key Concepts** for more information.
```

Comment:
Please change **Key Concepts** so that it is a clickable link to the section within the same document. Markdown docs allow creating hyperlinks to another section within the same document that starts with a header. For instance, if you have a section header named `### This is a header`, you can link to it through `[Link to header](#this-is-a-header)`.

"tag-convention" is an optional field. Allowed values are "incremental" or "floating" - refer to [**Key Concepts**](#key-concepts) for more information.


For more information, see the question and the top answer in https://stackoverflow.com/questions/2822089/how-to-link-to-part-of-the-same-document-in-markdown

![image](https://github.com/user-attachments/assets/2a20ea11-18a1-43e0-8482-4feda3b6ca67)